pkg/blackduck/crdhandler.go

```diff
@@ -114,19 +109,9 @@ func (h *Handler) ObjectCreated(obj interface{}) {
 // ObjectDeleted will be called for delete hub events
 func (h *Handler) ObjectDeleted(name string) {
 	log.Debugf("ObjectDeleted: %+v", name)
-
-	apiClientset, err := clientset.NewForConfig(h.kubeConfig)
-	crd, err := apiClientset.ApiextensionsV1beta1().CustomResourceDefinitions().Get("blackducks.synopsys.com", v1.GetOptions{})
```

Comment:
Same as Alert - Why deleting this?

Reply:
Namespace scope customer won't have access to CustomResourceDefinition permission.. CRD is at cluster scope...

Reply:
done